packages/fxa-settings/src/pages/ConnectAnotherDevice/en.ftl

```diff
@@ -0,0 +1,25 @@
+## Connect Another Device page
+# A user will only see this header if they are signed in. The header will be preceded by a green checkmark (rtl/ltr sensitive)
+connect-another-device-signed-in-header = You’re signed into { -brand-firefox }
+# A "success" message visible to users who verified via email
+connect-another-device-email-confirmed-banner = Email confirmed
+# A "success" message visible to users who verified via sign-in
+connect-another-device-signin-confirmed-banner = Sign-in confirmed
+# A message prompts the user to sign in to this instance of the Firefox browser so as to complete device sync. This is followed by a link labeled "Sign in"
+connect-another-device-signin-to-complete-message = Sign in to this { -brand-firefox } to complete set-up
```

Comment:
```suggestion
connect-another-device-signin-to-complete-message = Sign in to this { -brand-firefox } to complete setup
```
We use setup no hyphen for other Firefox products in US English, so best to align with that spelling.